2024-Wrapped.ipynb

```diff
@@ -0,0 +1,856 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Binärgewitter 2024\n",
+    "\n",
+    "Tagline?\n",
+    "Let's take a look:\n",
+    "Zuerst müssen wir alle librarys importieren und Daten laden."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import calmap"
```

Comment:
we should add a note to the readme to install calmap (https://github.com/Binaergewitter/stats/blob/main/README.md#getting-started)